### ISSUE TITLE
Rationalisation of dump files required

### ISSUE THREAD

**Issue by RichardWallis** [schemaorg/schemaorg#2304]
As identified in issues #2301 & #2302; within the definition download files, there is not full segregation between the terms defined in the core and terms defined in sections (extensions).

eg.The pending defined [CssSelectorType](https://schema.org/CssSelectorType), which is defined as the range for [cssSelector](https://schema.org/cssSelector) property, defined in the core vocab. As the _schema_ dump file only contains terms defined in the core, _CssSelectorType_ definition doesn't appear in that file.

Anecdotally, it appears that the main down of dump files are for '_schema_' (core only) the default offered on the [downloads page](https://schema.org/docs/developers.html#defs) and the '_all-layers_' which contains all definitions (including those in the _attic_ section).

I recommend that we rationalise the types of dump files offered (currently 8) down to 2 with the following contents:

- _schema_ - containing all sections except _attic_.
- _all-layers_ - containing all sections including _attic_.

**Comment by VladimirAlexiev** [comment]
#2537 shows that the current reduced `schema` lacks closure (some classes are referenced but not defined)

**Comment by danbri** [comment]
I agree. A few years ago we introduced a fair amount of engineering to support layered extension. In practice it turned out too difficult to disentangle various overlapping areas. Should we have an "extension" for Cultural Heritage? for Galleries / Archives / Libraries / Museums (GLAM)? Or different named efforts for bibliography and archival vocabulary? What about tourism, travel, real estate, and e-commerce? Where should ebooks be described? etc etc.

In practice we have gradually migrated back towards a simple structure.

 * Schema.org is Schema.org
   * It has an area called "attic" for things we did that we can't undo, but don't want to highlight
   * It has an area called "pending" for things we are doing, where we want to highlight a heightened potential for ongoing changes and the need for feedback

We have already downplayed the use of named subdomain extensions ("pending", "bib", etc.) in the site navigation. We have found little use for the defensive layering structure which attributed each triple in the schema definitions to one of those sections. In practice, "all layers" is the only sensible subset to use. There should be a simple triple representation, a consolidated Turtle representation (which looks as similar as possible to the MCF-flavoured Turtle source files), and I guess there is a desire for JSON-LD. Do we believe the CSV files have proved useful?

For naming I'd suggest something like 

 * "schemaorg-current" (everything minus attic)
 * "schemaorg-current-plus-removed" (everything including attic)

**Comment by RichardWallis** [comment]
Reviewing the files we currently produce I see we do not reference attic, ether individually or in other dump files.

As both _schema_.* and _all-layers_.* are often referenced in various issues and other comms, we ideally should not change those names without good reason.

To that end I suggest we retain the _schema_.* name, but update the contents to include all terms.  As _all-layers_.* is often used/recommended it should be retained for now, even though the contents will be identical to the updated _schema._* files.

The CSV versions have proved useful.  Many issues, and confusion about file contents, I have been involved with have often stemmed from someone reading a CSV versions.

**Comment by danbri** [comment]
> As both schema.* and all-layers.* are often referenced in various issues and other comms, we ideally should not change those names without good reason.

The good reason is that we no longer have a layered architecture for the sections of Schema.org.

Please use "schemaorg-current" and "schemaorg-all". This will make it easier to avoid confusing schema.ttl for (data/)schema.ttl, amongst other things. It is also developer friendly, in that it is a stronger reminder that this data comes from schema(.)org.

**Comment by ktk** [comment]
Will that influence download locations of URLs like `https://schema.org/version/latest/schema.nt` which is referenced here? https://schema.org/docs/developers.html#defs

**Comment by danbri** [comment]
@ktk we'll have to update the developer documents, yes. /cc @RichardWallis

**Comment by danbri** [comment]
see https://twitter.com/danbri/status/1283767294267731971 for announcement of this proposed change

**Comment by RichardWallis** [comment]
Implemented in PR #2654

**Comment by VladimirAlexiev** [comment]
Checked https://schema.org/version/latest/schemaorg-current-http.ttl, all used classes are defined:

```sparql
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
prefix schema: <http://schema.org/> 

select distinct ?class {
    ?p schema:domainIncludes|schema:rangeIncludes ?class
    filter not exists {?class a rdfs:Class}
} order by ?class
```
